Dockerfile

```diff
@@ -1,149 +1,179 @@
 ARG BUILD_FROM
-FROM ${BUILD_FROM}
-
-ARG \
-    BUILD_ARCH \
-    QEMU_CPU \
-    SSOCR_VERSION \
-    LIBCEC_VERSION \
-    PICOTTS_HASH \
-    TELLDUS_COMMIT
+####
+## Builder stage for ssocr, installs to /opt/ssocr
+FROM ${BUILD_FROM} AS ssocr-builder
+ARG SSOCR_VERSION
+ARG BUILD_FROM
+WORKDIR /tmp/
+RUN --mount=type=cache,target=/etc/apk/cache,sharing=locked,id=ssocr-builder-${BUILD_FROM}-${SSOCR_VERSION} \
```

Comment:
It's a nit, but I'd prefer to have `--mount` on another line - if there are multiple options, it makes them aligned and better readable.